test/onnx_import.cpp

```diff
@@ -195,6 +195,77 @@ TEST(onnx, model_conv2d_strides_assymetric_padding)
     EXPECT_EQ(expected_output, result.front());
 }
 
+TEST(onnx, model_average_pool_2d)
+{
+    // Pooling with strides=2 and no padding
+    auto model{ngraph::onnx_import::import_onnx_function(
+        ngraph::file_util::path_join(SERIALIZED_ZOO, "onnx/average_pool_2d.onnx"))};
+
+    // input data shape (1, 1, 4, 4)
+    Inputs inputs;
+    inputs.push_back(test::NDArray<float, 4>({{{{0.f, 1.f, 2.f, 3.f},
```

Comment:
`Inputs intpus{test::NDArray<float,4>{{{{{0.f, 1.f, }.get_vector()};`
use `{}` instead of `()` for object creation.